### DISCUSSION TITLE
Welcome to pyarmor Discussions!

### DISCUSSION THREAD

**Discussion by jondy** [dashingsoft/pyarmor#1016] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by mail-aleks** [comment]
Hi, Jondy!

Great program! Great job!

I have questions for you:
1. There is a 100% degree of security. what percentage - trial, what - basic, what - pro?
2. If I was able to successfully use the trial and I give the program to someone else, will it work for him indefinitely?
3. I am in Russia. Due to recent terrible events, I cannot buy as indicated on the site (mycommerce). is there another way to do this?

**Reply by mail-aleks** [reply]
can you send me a file to activate basic or pro in exchange for bitcoin?

**Comment by zerorzeror** [comment]
同级文件里，有11个py文件，需要批量对其加密，我尝试逐一对使用pyarmor.exe gen -e .30 ".\pyarmor\???.py" ，对11个文件加密，发现只有1个文件生效，其余都报错，我猜想应该是我命令参数用错了，请问应该使用什么参数？

**Reply by wjl130825** [reply]
PyInstaller\loader\pyimod02_importers.py", line 384, in exec_module
  File "pyarmor_runtime_005595\__init__.py", line 2, in <module>
RuntimeError: [WinError 10054] 远程主机强迫关闭了一个现有的连接。 (1:11076)现在打包，运行软件提示这个问题，啥原因
